cluster/minio-tenant.go

```diff
@@ -52,7 +52,7 @@ func mkTenantMinioContainer(sgTenant *StorageGroupTenant, sgNode *StorageGroupNo
 
 	tenantContainer := v1.Container{
 		Name:  fmt.Sprintf("%s-minio-%d", sgTenant.Tenant.ShortName, sgNode.Num),
-		Image: "minio/minio:RELEASE.2019-12-17T23-16-33Z",
```

Comment:
can we just continue with the latest release until the new one comes? else edge could change.

Reply:
New Release is there, please move to that one `RELEASE.2019-12-19T22-52-26Z`

Reply:
done